src/jtagmkII.c

```diff
@@ -3601,6 +3601,37 @@ static int jtagmkII_flash_clear_pagebuffer32(const PROGRAMMER *pgm) {
     return -1;
 }
 
+/*** Periodic calls in terminal mode to keep `JTAGMKII_UPDI` enabled. ***/
```

Comment:
Slightly unusual way to write comments in AVRDUDE; the project has taken to one-line comments like so
```
// Periodic calls in terminal mode to keep the programmer jtagmkii_updi enabled
```

(no end punctuation, no quotes, sentence capitalisation)